python/src/GaussianProcessConditionalCovariance_doc.i.in

```diff
@@ -0,0 +1,225 @@
+%feature("docstring") OT::GaussianProcessConditionalCovariance
+"Conditional covariance post processing of a Gaussian Process Regression result.
+
+    GaussianProcessConditionalCovariance(*gprResult*)
+
+
+
+Parameters
+----------
+gprResult :  :class:`~openturns.experimental.GaussianProcessRegressionResult`
+    The result class of a gaussian process regression.
+
+
+Notes
+-----
+The class provides services around conditional covariance of a Gaussian Process Regression
+
+
+Examples
+--------
+Create the model :math:`\cM: \Rset \mapsto \Rset` and the samples:
+
+>>> import openturns as ot
+>>> from openturns.experimental import GaussianProcessRegression
+>>> from openturns.experimental import GaussianProcessConditionalCovariance
+>>> trend = ot.SymbolicFunction(['x'],  ['1'])
+>>> sampleX = [[1.0], [2.0], [3.0], [4.0], [5.0], [6.0]]
+>>> sampleY = trend(sampleX)
+
+Create the algorithm:
+
+>>> covarianceModel = ot.SquaredExponential([1.0])
+>>> covarianceModel.setActiveParameter([])
+
+>>> algo = GaussianProcessRegression(sampleX, sampleY, covarianceModel, trend)
+>>> algo.run()
+>>> result = algo.getResult()
+>>> condCov = GaussianProcessConditionalCovariance(result)
+>>> c = condCov([1.1])
+
+"
+
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::GaussianProcessConditionalCovariance::getConditionalMean
+"Compute the conditional mean of the Gaussian process on a point or a sample of points.
+
+Available usages:
+    getConditionalMean(x)
+
+    getConditionalMean(sampleX)
```

Comment:
I would remove this and simply state that the parameter can be either a sequence of float or a 2-d sequence of float (and explain how this changes the bahavior).

Same for other accessors in this class.